index.html

```diff
@@ -896,8 +899,10 @@ <h4>parseDerivedProofValue</h4>
           <ol class="algorithm">
             <li>
 Ensure the `proofValue` string starts with
-<span class="codepoint" translate="no"><bdi lang="en">u</bdi>
-<code class="uname">U+0055 LATIN SMALL LETTER U</code></span>, indicating that
+<span class="codepoint" translate="no">
+  <bdi lang="en"><code title="LATIN SMALL LETTER U">u</code></bdi>
+  (<code class="codepoint">U+0075</code>,
+  <code class="uname">LATIN SMALL LETTER U</code>)</span>, indicating that
 it is a multibase-base64url-no-pad-encoded value, and throw an error if it does
```

Comment:
To match the earlier...
```suggestion
it is a `multibase-base64url-no-pad-encoded` value, and throw an error if it does
```